modular_skyrat/modules/stories_system/code/helping_code/outfits.dm

```diff
@@ -174,8 +174,37 @@
 	ears = /obj/item/radio/headset/headset_med
 	shoes = /obj/item/clothing/shoes/sneakers/white
 	l_hand = /obj/item/clipboard
+
 	back = /obj/item/storage/backpack/satchel/med
 	backpack_contents = list(
 		/obj/item/pen = 1,
 		/obj/item/paper = 2,
 	)
+
+/datum/outfit/nri_shore_leave
+	name = "Shore Leave NRI Marine"
+
+	glasses = /obj/item/clothing/glasses/sunglasses
+	ears = /obj/item/radio/headset/guild
+	uniform = /obj/item/clothing/under/costume/nri
+	suit = /obj/item/clothing/suit/armor/vest/russian/nri
+	gloves = /obj/item/clothing/gloves/combat
+	shoes = /obj/item/clothing/shoes/combat
+	belt = /obj/item/storage/belt/military/nri
+
+	back = /obj/item/storage/backpack/duffelbag/syndie/nri
+	backpack_contents = list(
+		/obj/item/storage/box/nri_survival_pack = 1,
+		/obj/item/crucifix = 1,
+		/obj/item/ammo_box/magazine/m9mm_aps = 1,
+		/obj/item/modular_computer/tablet/pda/security = 1,
+	)
+
+	l_pocket = /obj/item/gun/ballistic/automatic/pistol/ladon/nri
```

Comment:
I'd prefer not to give them anything stronger than a Makarov; two-burst branded APS might be a little bit too much.